src/apps/components/itemSheetsHeader/ItemSheetHeader.svelte

```diff
@@ -54,22 +54,80 @@
             </div>
         {/if}
     </div>
-
-    {#if $item.system.broken}
-        <i
-            class="broken-item-icon fa-solid fa-heart-crack"
-            data-tooltip="A5E.BrokenItem"
-            data-tooltip-direction="DOWN"
+    <div class="button-container">
+        <!-- {#if game.user.isGM} -->
+        <button
+            class="action-button fa-solid fa-circle-question"
```

Comment:
`action-button` should be replaced with something that make senses here like `header-button`